New feature, exclude tables that match a regular expression.

jamidon opened the pull request:
I use this app a lot when creating restify+postgresql REST services - works great, thanks. This pull request adds a new feature to exclude table definitions being generated for table names that match a list of regular expressions. 

I use it to exclude PostGIS tables and partition tables that I never use. The antonym to this feature would be to include a feature to only include those tables that match a list of expressions. Not something that I need but if you feel this would be useful, I can add it too.

jamidon committed:
Added feature that allows excluding tables that match one or more regular expressions.

jamidon committed:
Error checking for regular expressions.

jamidon committed:
Added test for regular expression.

jamidon committed:
Doc update for --exclude-regex.

jamidon committed:
Corrected problems with earlier nodejs versions.

jamidon committed:
Corrected problems with earlier nodejs versions.

tmont commented:
Definitely a useful feature, thanks! I fixed a few formatting issues (sorry, I love TABs) and squashed it in. It's been published to npm as version `1.5.0`.

Thanks again!

jamidon commented:
This got buried in my email...

Thanks. As far as the formatting goes, I had a hard time replicating what
you did so glad it worked out.

-- jda

On Feb 1, 2017 2:02 AM, "Tommy Montgomery" <notifications@github.com> wrote:

> Definitely a useful feature, thanks! I fixed a few formatting issues
> (sorry, I love TABs) and squashed it in. It's been published to npm as
> version 1.5.0.
>
> Thanks again!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tmont/node-sql-generate/pull/6#issuecomment-276587142>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbkJrMXEjFwslwQFSGVwXspFOAL9teRks5rYC3-gaJpZM4Lwlb8>
> .
>
